db/version_set_test.cc

```diff
@@ -869,6 +869,27 @@ class VersionSetTestBase {
     mutex_.Unlock();
   }
 
+  void CreateColumnFamily(const std::string& cf_name,
+                          const ColumnFamilyOptions& cf_options,
+                          ColumnFamilyData*& cfd) {
```

Comment:
How about returning the `ColumnFamilyData` pointer instead of using an out param?